vispy/visuals/mesh.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, vertices=None, faces=None, vertex_colors=None,
         Visual.__init__(self, **kwargs)
         
         self.set_gl_state('translucent', depth_test=True,
-                          cull_face='front_and_back')
+                          cull_face=False)
```

Comment:
Originally these were set to 'front_and_back', which should cause both faces to be _culled_. However, a separate bug prevented that being set correctly. After I fixed that bug, the meshes mysteriously stopped drawing :)